index.js

```diff
@@ -49,7 +49,12 @@ module.exports = {
     // TODO: In order to make the addon work in EmberTwiddle, we cannot use // the `tests` prop til
     // https://github.com/joostdevries/twiddle-backend/pull/28 is merged.
     // return !!this.app.tests;
-    return this.app.env !== 'production';
+
+    if('EMBER_CLI_FASTBOOT' in process.env) {
```

Comment:
I think it would fail when:
```
EMBER_CLI_FASTBOOT=false ember s
```
can we change this condition to check if the value is true, something like `process.env.EMBER_CLI_FASTBOOT === 'true'`?